_posts/2017-08-22-visdat.md

```diff
@@ -0,0 +1,155 @@
+---
+name: visdat
+layout: post
+title: "Onboarding `visdat`, a tool for preliminary visualisation of whole dataframes"
+date: 2017-08-22
+authors:
+  - name: Nicholas Tierney
+    url: http://www.njtierney.com
+categories:
+  - blog
+tags:
+- R
+- software
+- package
+- review
+- community
+- visdat
+- data-visualisation
+---
+
+> Take a look at the data
+
+This is a phrase that comes up when you first get a dataset.
```

Comment:
All mentions of visdat should be in backticks for consistent formatting: `visdat`